Check out my advice for newbies.

Elysehmyers opened the pull request.

redrambles commented:
Hey @Elysehmyers !

This is close! If you take a look at your [change set here](https://github.com/skillcrush/105-practice/pull/164/files) - you'll see that you have one file `advice/elyse-myers-advice` - that doesn't have an .html extension - which you can remove.

Once you've done that, you'll want to make sure to add a link in the main index.html file your advice page from the main index.html file, so that folks can find your words of wisdom, once your code is merged in. If you're not sure how to do this, feel free to open the index.html file and take a look at other students' links to see how you can add one of your own!

Once you add your link, add and commit your changes and then push your branch up once more, which will automatically update your pull request. Then we should be good to merge in your code!

Let us know if you have any questions!

Cheers,
Ann

Elysehmyers commented:
Thank you for your feedback! I added my advice into the main index.html so others can access my page. I also went ahead and deleted that duplicate file that was missing the .html extension!

acodeinprogress commented:
Hey @Elysehmyers,

Love your advice!! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon
